apps/judicial-system/api/src/app/modules/case/case.resolver.ts

```diff
@@ -57,6 +57,24 @@ export class CaseResolver {
     )
   }
 
+  @Query(() => [Case], { nullable: true })
+  async connectedCases(
+    @Args('input', { type: () => CaseQueryInput })
+    input: CaseQueryInput,
+    @CurrentGraphQlUser()
+    user: User,
+    @Context('dataSources') { backendApi }: { backendApi: BackendApi },
+  ): Promise<Case[]> {
+    this.logger.debug('Getting connected cases')
+
+    return this.auditTrailService.audit(
+      user.id,
+      AuditedAction.GET_CASES,
```

Comment:
Is this the appropriate audited action here?

Reply:
Good point, I've created a new one GET_CONNECTED_CASES